src/SFA.DAS.EAS.Web/Orchestrators/Mappers/ApprenticeshipMapper.cs

```diff
@@ -38,11 +39,6 @@ public class ApprenticeshipMapper : IApprenticeshipMapper
             ILog logger,
             IAcademicYearValidator academicYearValidator)
         {
-            if (hashingService == null) throw new ArgumentNullException(nameof(hashingService));
-            if (currentDateTime == null) throw new ArgumentNullException(nameof(currentDateTime));
-            if (mediator == null) throw new ArgumentNullException(nameof(mediator));
-            if (academicYearValidator== null) throw new ArgumentNullException(nameof(academicYearValidator));
```

Comment:
Curious about the removal of these guards. Some classes check for null injectables in the c'tor, and some don't. Should we be consistent?